packages/terra-search-field/src/SearchField.jsx

```diff
@@ -88,8 +89,18 @@ class SearchField extends React.Component {
 
     return (
       <div {...customProps} className={searchFieldClassNames}>
```

Comment:
I'll probably log a new issue to wrap this (and date-picker and time-input) with a Field component since several new props (label, error, help, isInline, required) will need to be added and that is out of the scope of this PR.